omod/src/main/webapp/resources/partials/daterangepicker.html

```diff
@@ -1,13 +1,14 @@
-<div class="date-range-picker-div" class="inlineBox">
-    <p> {{headermessage}} </p>
-    <span class="angular-datepicker">
-        <input class="appoinmentrangepicker" type="text" is-open="startDateOptions.opened" ng-model="startDate" min="{{startDateMin}}" max="endDate" show-weeks="false" datepicker-popup="dd MMM yyyy" readonly/>
-        <i class="icon-calendar small add-on" ng-click="startDateOptions.open($event)" ></i>
-        <a class="date-range-picker-clear-link add-on" ng-click="startDateOptions.clear($event)">{{ clearLinkText }}</a>
+<div class = "col-xs-5">
+  <div class="date-range-picker-div">
+  <p>{{headermessage}}&nbsp;<span><a class="date-range-picker-clear-link add-on1" ng-click="startDateOptions.clear($event)">{{ clearLinkText }}</a></span></p>
```

Comment:
i dont think we would need to shift that anchor tag out of the span element. rather you could use bootstrap columns or some thing else to format the elements.

Reply:
Formatting would be also great, but again the first design /placement of clear buttons were a bit challenging,Using bootstrap columns would be also great but am talking about clear link. i will do it using bootstrap, and i hope to ping you